### DISCUSSION TITLE
如何按时间范围与saas的租户ID联合起来做分区

### DISCUSSION THREAD

**Discussion by feiver** [apache/doris#8593] [A - General / Q&A]
如题：saas系统租户隔离，如何使用时间范围与saas的租户ID做分区

**Comment by dataalive** [comment]
If it is in terms of performance，you could just use like [https://doris.apache.org/getting-started/data-partition.html#data-division](https://doris.apache.org/getting-started/data-partition.html#data-division)
```
PARTITION BY RANGE(`date`) 
```
for table schema ,put user id in the first term , because Doris use sorted table.

or for security reasons,  try search Multi-column partition to see whether it helps
